client/scripts/com/monsters/maproom_advanced/MapRoomCell.as

```diff
@@ -15,7 +15,8 @@ package com.monsters.maproom_advanced
    public class MapRoomCell extends MapRoomCell_CLIP implements IMapRoomCell
    {
        
-      
+      internal var _cell_id:int;
```

Comment:
Reminder: Remove any new client side logic, we should emulate the original server.